README.md

```diff
@@ -34,20 +34,23 @@ See: https://b3n.org/automatic-ripping-machine
 
 ## Install
 
+    sudo apt-get install git
+    sudo apt-get install regionset
+    sudo regionset /dev/sr0
```

Comment:
This is good, I think there should be a note warning that the region can only be changed 5? times on most DVD players then it gets stuck.  Also might mention if the region is already set there's no need to run it.